### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Denton-L**

### CODE REVIEW

File: docs/SFTPCopy-action.md [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
-In order perform SFTP copy, we require host and port on which the SFTP server is running. SFTP implements secure file
-transfer over SSH. Typically port number 22 is used for SFTP(which is also default port for SSH). We also require valid
-credentials in the form of user name and password. Please make sure that you are able to SSH to the SFTP server using
-specified user and password. SSH connection to SFTP server can be customized by providing additional configurations
-such as enable host key checking by setting configuration property 'StrictHostKeyChecking' to 'yes'. These additional
-configurations can be specified using `Properties for SSH` section.
-
-Directory on the SFTP server which needs to be copied can be specified using `Source directory` property. The specified
-directory should exist and absolute path to the directory must be provided. If directory is empty then execution will
+In order to perform a SFTP copy, we require the host and port on which the SFTP server is running. SFTP implements secure file
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: an SFTP

File: docs/SFTPCopy-action.md [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
+transfer over SSH. Typically, port number 22 is used for SFTP (which is also default port for SSH). We also require valid
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: also the default port for SSH

File: docs/SFTPCopy-action.md [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
+credentials in the form of a username and password. Please make sure that you are able to SSH to the SFTP server using
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: Valid credentials in the form of a username and password are required.

File: docs/SFTPCopy-action.md [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
+the specified username and password. The SSH connection to SFTP server can be customized by providing additional configurations
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: to the SFTP server can customized with options such as host key checking by setting...Additional configurations can be specified in the `Properties for SSH`...

File: docs/SFTPCopy-action.md [hunk 5]
```diff
@@ -1,51 +1,51 @@
...
+such as enabling host key checking by setting the configuration property `StrictHostKeyChecking` to `yes`. These additional
+configurations can be specified in the `Properties for SSH` section.
+
+The directory on the SFTP server which needs to be copied can be specified using the `Source directory` property. The specified
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: The directory to be copied from the SFTP server can be set using ...

File: docs/SFTPCopy-action.md [hunk 6]
```diff
@@ -1,51 +1,51 @@
...
+directory should exist and absolute path to the directory must be provided. If directory is empty then the execution will
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: The directory should exist and an absolute path to it must be provided.

Comment (awholegunch) [inline docs/SFTPCopy-action.md]: If the directory is empty, execution will continue without error.

File: docs/SFTPCopy-action.md [hunk 7]
```diff
@@ -1,51 +1,51 @@
...
 
-Files from the SFTP server can optionally be uncompressed while copying to HDFS. Currently uncompress option is only supported
-for the zip files.
+Files from the SFTP server can, optionally, be uncompressed while being copied to HDFS. Currently the uncompress option is only supported
+for zip files.
 
-Typically SFTP server acts as a temporary storage for the files and once processed the files can be deleted. Comma
-separated list of file names on the SFTP server which were copied to HDFS during the current run, is stored in a
-variable named `sftp.copied.file.names`. SFTP Delete action can be configured to run at the end of the pipeline,
+Typically, SFTP server acts as temporary storage for the files and, once processed, the files can be deleted. A
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: Typically, the SFTP server...in the variable `sftp.copied.file.names`.

File: docs/SFTPCopy-action.md [hunk 8]
```diff
@@ -1,51 +1,51 @@
...
-<a href="https://cdap-users.herokuapp.com/"><img alt="Join CDAP community" src="https://cdap-users.herokuapp.com/badge.svg?t=sftp-actions"/></a>
-[![Build Status](https://travis-ci.org/hydrator/sftp-actions.svg?branch=develop)](https://travis-ci.org/hydrator/sftp-actions) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) <img src="https://cdap-users.herokuapp.com/assets/cdap-action.svg"/>
+[![Join CDAP community](https://cdap-users.herokuapp.com/badge.svg?t=sftp-actions)](https://cdap-users.herokuapp.com)
+[![Build Status](https://travis-ci.org/hydrator/sftp-actions.svg?branch=develop)](https://travis-ci.org/hydrator/sftp-actions)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![CDAP Action](cdap-users.herokuapp.com/assets/cdap-action.svg)](http://docs.cask.co/cdap)
 
-
-SFTP copy allows copying of the files from the specified directory on SFTP servers and write them to HDFS as the destination.
+SFTP Copy allows copying of the files from the specified directory on SFTP servers and write them to HDFS as the destination.
```
Comment (awholegunch) [inline docs/SFTPCopy-action.md]: SFTP Copy copies files from a specified directory on an SFTP server and writes them to a destination in HDFS.

File: README.md [hunk 9]
```diff
@@ -23,15 +25,21 @@ These files can be used to deploy your plugins.
...
+```
 
 For example, if your artifact is named 'sftp-actions-1.0.0':
 
+```
     > load artifact target/sftp-actions-1.0.0.jar config-file target/sftp-actions-1.0.0.json
+```
 
-## Mailing Lists
+Mailing Lists
```
Comment (awholegunch) [inline README.md:41]: What did we say about these heads? Did we decide on rst or markdown style?

File: docs/SFTPDelete-action.md [hunk 10]
```diff
@@ -1,34 +1,34 @@
...
 CDAP Action plugin to delete the specified files from the SFTP server.
 
-
 Use Case
 --------
-This plugin is generally used along with the SFTP copy plugin. SFTP copy plugin copies the files from SFTP
-server to the desired location such as `HDFS`. Once copied to `HDFS`, files can be processed by pipeline.
-When the pipeline is successful, the files from SFTP server can be deleted. SFTP delete plugin can be used in
+
+This plugin is generally used along with the SFTP Copy plugin. The SFTP Copy plugin copies the files from the SFTP
```
Comment (awholegunch) [inline docs/SFTPDelete-action.md]:
```
This plugin is often used with the [SFTP Copy plugin](SFTPCopy-action.md), which copies files from an SFTP
server to a specified location such as `HDFS`. Once copied to `HDFS`, files can then be processed by a pipeline.

File: docs/SFTPDelete-action.md [hunk 11]
```diff
@@ -1,34 +1,34 @@
...
+server to the desired location, such as `HDFS`. Once copied to `HDFS`, files can be processed by pipeline.
+When the pipeline is successful, the files from SFTP server can be deleted. The SFTP Delete plugin can be used at
```
Comment (awholegunch) [inline docs/SFTPDelete-action.md]: the files from the SFTP server

File: docs/SFTPDelete-action.md [hunk 12]
```diff
@@ -1,34 +1,34 @@
...
-| **Password** | **Y** | N/A | Specifies the password of the user.|
-| **Files to be deleted** | **Y** | ${sftp.copied.file.names} | Comma separated list of files on the SFTP server to be deleted. Default value for this field is a Macro which will be substituted by SFTP copy plugin when this plugin is used with it. |
-| **Destination Directory** | **Y** | N/A | Destination directory on the file system, where files need to be copied. If directory does not exist, it will lbe created.|
+| :------------ | :------: | :------ | :---------- |
+| **Host** | **Y** | N/A | Specifies the host name of the SFTP server. |
+| **Port** | **N** | 22 | Specifies the port on which SFTP server is running. |
+| **User** | **Y** | N/A | Specifies the name of the user which will be used to connect to the SFTP server. |
+| **Password** | **Y** | N/A | Specifies the password of the user. |
+| **Files to be deleted** | **Y** | ${sftp.copied.file.names} | Comma separated list of files on the SFTP server to be deleted. Default value for this field is a Macro which will be substituted by SFTP Copy plugin when this plugin is used with it. |
```
Comment (awholegunch) [inline docs/SFTPDelete-action.md]: Comma-separated

File: docs/SFTPDelete-action.md [hunk 13]
```diff
@@ -1,34 +1,34 @@
...
+| **Destination Directory** | **Y** | N/A | Destination directory on the file system, where files need to be copied. If directory does not exist, it will lbe created. |
 | **Continue execution on error** | **N** | false | Boolean flag to determine whether to proceed with next files in case there is a failure in deletion of any particular file. |
-| **Properties for SSH** | **N** | N/A | Specifies the properties that are used to configure SSH connection to the FTP server. For example to enable verbose logging add property 'LogLevel' with value 'VERBOSE'. To enable host key checking set 'StrictHostKeyChecking' to 'yes'. SSH can be configured with the properties described here 'https://linux.die.net/man/5/ssh_config'. |
-
+| **Properties for SSH** | **N** | N/A | Specifies the properties that are used to configure SSH connection to the FTP server. For example, to enable verbose logging, add property `LogLevel` with value `VERBOSE`. To enable host key checking, set `StrictHostKeyChecking` to `yes`. SSH can be configured with the properties described here 'https://linux.die.net/man/5/ssh_config'. |
```
Comment (awholegunch) [inline docs/SFTPDelete-action.md]: add the property `LogLevel` with the value

Comment (awholegunch) [inline docs/SFTPDelete-action.md]:
```
SSH can be configured with [these properties](https://linux.die.net/man/5/ssh_config).

**Review by awholegunch — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (Denton-L) [reply README.md:41]: I don't think we ever decided on anything, but I've been using RST in the meantime for consistency with the other PRs I've submitted.

**Pull request closed without merging**